solutions/silver/cf-51C.mdx

```diff
@@ -0,0 +1,73 @@
+---
+id: cf-51C
+source: CF
+title: Three Base Stations
+author: Kenneth Karaisz Jr.
+---
+
+## Explanation
+
+Let's binary search on the answer.
```

Comment:
can you add hints

Reply:
Any directions on doing so, or an example solution page somewhere?

Reply:
look in my recent solution PR, i think that one has hints